jac-uk/admin
# #1952 Changes to Equality and diversity Form

## What's included?

Some social mobility questions on the application form need replacing after 01-04-2023. They are:

What was the occupation of your main household earner when you were about aged 14?
Which type of school did you attend for the most time between the ages of 11 and 16?
Did either of your parents attend university and gain a degree (e.g. BA/BSc or equivalent) by the time you were 18?

These questions should be replaced in the following areas.
Exercise. Update application view with new social mobility questions
Exercise. Update diversity report with new social mobility questions
Exercise. Update custom report with new social mobility questions
Exercise. Update dashboard with new social mobility questions
Exercise. Update downloads with new social mobility questions
Exercises. Update exercises downloads to include both sets of questions (as for quite a while there will be a mix of exercises from before and after the question changes)

Closes #1952

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Ensure that the following Digital Platform branch is used when testing: origin/feature/admin-1952-diversity-changes
Ensure that the questions appear in the areas of the site above for exercises with an application open date after 01-04-2023.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
